exovetter/const.py

```diff
@@ -29,4 +29,8 @@
 """PPM"""
 
 frac_amp = u.dimensionless_unscaled
-"""Frac amp"""
+
+string_to_offset = {'bkjd': bkjd, 'kjd': bkjd,
+                    'btjd': btjd,
+                    'bjd': bjd
+                    }
```

Comment:
Unless it is causing problems, I don't see any reason to remove it.

Reply:
It introduces maintenance burden.

Reply:
Let's leave it for the moment. I actually think we will eventually find we need to expand it.  Astronomers are just not good at being consistent with these names and we should deal with it for them.  But I could be wrong here.

Reply:
I am not a fan of enabling bad habits, but if you want this anyway, may I propose a helper function instead of a dictionary?

Reply:
I agree that it is a bit klunky as is...but can I suggest that we leave this for a future pull request where you and @fergalm can discuss what's the right thing to do.  I don't see the advantage of a dictionary over a function, but I also don't see it as the tall tent pole right now.  

We have little control over what other programs or telescopes decide to use for these names, there isn't much of a standard. So someone could decide to label it as 'TJD' in their high-level science product instead of 'BTJD' so at some point we need to tell this code that these are the same. I'd rather not force the user to figure that out when we already know the answer.

Reply:
If you want it to be configurable, then perhaps https://docs.astropy.org/en/latest/config/index.html#using-astropy-config but yes, we can revisit in the future.

Can you please at least at a docstring so it gets documented? Thanks.

Reply:
Will do.  I assume it should be in that format with the three quotes that goes below the variable like you did for the rest.

Reply:
Yes. 😄